articles/clients/client-settings/native/index.md

```diff
@@ -1,24 +1,20 @@
 ---
-description: Describes the differences between the Auth0 Client types and what they mean when it comes to settings
+description: Client settings for Native
+url: /clients/client-settings/native
+toc: true
 ---
 
-# Client Settings
+# Client Settings: Native
 
 When creating an Auth0 Client, you'll be asked to indicate the *type* of Client you want to create. 
 
 ![Window for selecting client type](/media/articles/clients/create-clients.png)
 
-You can choose from one of four options:
-
-* **Native**: used for desktop or mobile apps that run natively on the the device
-* **Single Page Web Applications**: apps with a JavaScript front-end utilizing an API
-* **Regular Web Applications**: traditional web apps with refresh behavior
-* **Non Interactive Clients**: CLIs, daemons, or services running on the back-end
+For desktop or mobile apps running natively on the device, you'll want to create a Native Client.
 
 ## Settings
 
-By default, the [settings](${manage_url}/#/clients/${account.clientId}/settings) include the following:
-
+By default, the [settings](${manage_url}/#/clients/${account.clientId}/settings) for all Client types include the following:
 
 - **Name**: The name of your client. This information is editable and you will see in the portal, emails, logs, and so on.
```

Comment:
I think you should use doc fragments for the settings all clients have in common.